test/parallel/test-fs-promises.js

```diff
@@ -82,24 +82,44 @@ function verifyStatObject(stat) {
     stats = await stat(dest);
     verifyStatObject(stats);
 
+    stats = await handle.stat();
+    verifyStatObject(stats);
+
     await fdatasync(handle);
+    await handle.datasync();
     await fsync(handle);
+    await handle.sync();
 
     const buf = Buffer.from('hello world');
-
     await write(handle, buf);
-
     const ret = await read(handle, Buffer.alloc(11), 0, 11, 0);
     assert.strictEqual(ret.bytesRead, 11);
     assert.deepStrictEqual(ret.buffer, buf);
 
+    const buf2 = Buffer.from('HELLO WORLD');
+    await handle.write(buf2);
+    const ret2 = await handle.read(Buffer.alloc(11), 0, 11, 0);
+    assert.strictEqual(ret2.bytesRead, 11);
+    assert.deepStrictEqual(ret2.buffer, buf);
```

Comment:
Shouldn't this be compared with `buf2`?

Reply:
Nit: Also, if we reduce magic numbers (like 11 in this case) it would be better.